app/workers/mapping/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""
```

Comment:
Bit of a niggly one, but could I suggest we rename this project? `mapping` is not all that descriptive of what the `MappingRules` function does, and the name `mapping` is already used as the base Django app.

Or is that the whole point, and it's important to reuse the same name to pick up the base app?

Reply:
Yes it is the whole point - as we now share the database models, they have a dependency on a `mapping` app existing in the Django project.

Reply:
Gotcha!